cloudbuild.yaml

```diff
@@ -0,0 +1,50 @@
+steps:
+  - name: gcr.io/cloud-builders/docker
```

Comment:
Is it intentional that all the entries in this 'steps' list have identical names?